srcpkgs/taskd/INSTALL

```diff
@@ -2,9 +2,8 @@
 case "$ACTION" in
 post)
 	if [ "$UPDATE" != "yes" ]; then
-		taskd init --data var/lib/taskd
-		taskd config --data var/lib/taskd log var/log/taskd.log
-		chown -R taskd:taskd var/lib/taskd
+		taskd init --data /var/lib/taskd --log=-
+		chown -R taskd:taskd /var/lib/taskd
```

Comment:
One thing, I'm not sure if changing `var/lib/taskd` to `/var/lib/taskd` is a right thing to do.

Reply:
Should be relative, INSTALL scripts can be run with --rootdir option

Reply:
Reverted it. For some reason the config `/var/lib/taskd` is not updated (no proper log option) when path is relative.
Any thoughts?

Reply:
`/var/lib/taskd/config` is
```
confirmation=1
extensions=/usr/libexec/taskd
ip.log=on
log=var/log/taskd.log
pid.file=/tmp/taskd.pid
queue.size=10
request.limit=1048576
root=var/lib/taskd
server=
trust=strict
verbose=1
```
instead of
```
confirmation=1
extensions=/usr/libexec/taskd
ip.log=on
log=-
pid.file=/tmp/taskd.pid
queue.size=10
request.limit=1048576
root=var/lib/taskd
server=
trust=strict
verbose=1
```
See the `log` option